### DISCUSSION TITLE
Sequoia 15.1

### DISCUSSION THREAD

**Discussion by danizagra** [will-stone/browserosaurus#702] [Q&A]
I hope this message finds you well. I am writing to report an issue with Browserosaurus, which seems to have stopped functioning correctly after updating to Sequoia 15.1.

The app no longer opens or functions as expected when attempting to select a browser. I’ve tried basic troubleshooting steps such as reinstalling the app and restarting my device, but the issue persists.

If this is a known issue or if there are any additional steps I should try, I would appreciate your guidance. Thank you for your support and for developing such a great tool!

Looking forward to your response.

**Comment by will-stone** [comment]
Hi. Sorry to hear that, please fill out an issue and we can discuss it there.
